poi-ooxml/src/main/java/org/apache/poi/xwpf/usermodel/XWPFRun.java

```diff
@@ -1505,7 +1505,7 @@ public void setTextHighlightColor(String colorName) {
      */
     @Deprecated
     @Removal(version = "7.0.0")
-    public STHighlightColor.Enum getTextHightlightColor() {
+    public STHighlightColor.Enum getTextHighlightColor() {
```

Comment:
we've left this misspelled method there and deprecated it - `getTextHighlightColor()` exists already - so please remove this change